include/ignition/gazebo/Util.hh

```diff
@@ -169,6 +169,33 @@ namespace ignition
         const Entity &_entity,
         const EntityComponentManager &_ecm);
 
+    /// \brief Helper function to "enable" a component (i.e. create it if it
+    /// doesn't exist) or "disable" a component (i.e. remove it if it exists).
+    /// \param[in] _ecm Mutable reference to the ECM
+    /// \param[in] _entity Entity whose component is being enabled
+    /// \param[in] _enable True to enable (create), false to disable (remove).
+    /// \return True if a component was created or removed, false if nothing
+    /// changed.
+    template <class ComponentType>
+    bool IGNITION_GAZEBO_VISIBLE enableComponent(EntityComponentManager &_ecm,
+        Entity _entity, bool _enable)
```

Comment:
```suggestion
        Entity _entity, bool _enable = true)
```
Sorry I should probably have suggested this yesterday, but I think it'd be nice to have this as in most cases in systems we enable components. Feel free to refute this suggestion.

Reply:
Ahhh I thought the same :smile: Will do :+1: